aws/resource_aws_vpc.go

```diff
@@ -33,7 +33,7 @@ func resourceAwsVpc() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     true,
-				ValidateFunc: validateCIDRNetworkAddress,
+				ValidateFunc: validateVPCCIDRNetworkAddress,
```

Comment:
We should use the available Terraform provider SDK validation function instead of implementing our own. 😄 
 https://godoc.org/github.com/hashicorp/terraform/helper/validation#CIDRNetwork

https://github.com/hashicorp/terraform/blob/9ada53547faa5b913c8b95d07bb47c0dbad476c5/helper/validation/validation.go#L204-L236

```suggestion
				ValidateFunc: validation.CIDRNetwork(16, 28),
```

Reply:
Thanks @bflad 👍